release-please-config.json

```diff
@@ -5,10 +5,8 @@
     "apps/hobbies-helsinki":{ "release-type": "node" },
     "apps/events-helsinki": { "release-type": "node" },
     "apps/sports-helsinki": { "release-type": "node" },
-    "packages/eslint-config-bases": { "release-type": "node" },
-    "packages/components": { "release-type": "node" },
-    "packages/common-i18n": { "release-type": "node" },
-    "proxies/events-graphql-federation": { "release-type": "rust", "package-name": "federation-router" }
+    "proxies/events-graphql-federation": { "release-type": "rust", "package-name": "federation-router" },
```

Comment:
The Apollo-Router is done with rust, but this folder does not actually have any executable, except the dockerfile. The filetypes which there are at the moment are:

- .graphql, for the schemas
- .yaml for all the router app configuration
- .md for readmes
- Dockerfile

I don't know if it really matters and that is the only reason I even mention this 😄. I think the rust is a release-type here, *if it does not have any side-effects!*

Reply:
Yeah, it doesn't really matter here, since `package-name` is defined - `release-type` is just for the bot to know which files to modify. But actually good point - this is just bunch of config files, used with the binary written in rust. 

The `release-type` could be also `simple` which would just write changelog and a version.txt. So I think it would be better to change this to `simple`, since the release essentially contains just config files for the actual router to run; ie . there won't be any rust specific binary etc. included in the release; `rust` _would_ basically need `Cargo.toml` to exists in the dir anyway, which this doesn't include.